File: walletsdk/src/main/java/leaf/prod/walletsdk/model/order/Erc1400Params.java
/**
 * Created with IntelliJ IDEA.
 * User: kenshin wangchen@loopring.org
 * Time: 2018-11-16 11:34 AM
 * Cooperation: loopring.org 路印协议基金会
 */
package leaf.prod.walletsdk.model.order;

import java.io.Serializable;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Erc1400Params implements Serializable {

    private Integer tokenStandardS;

    private Integer tokenStandardB;

    private Integer tokenStandardFee;

    private String trancheS;

    private String trancheB;

    private String transferDataS;
}

File: loopr-ios/Models/Order/ERC1400Params.swift
//
//  Depth.swift
//  loopr-ios
//
//  Created by kenshin on 2018/3/6.
//  Copyright © 2018年 Loopring. All rights reserved.
//

import Foundation

class ERC1400Params {

    var tokenStandardS: Int = 0

    var tokenStandardB: Int = 0

    var tokenStandardFee: Int = 0

    var trancheS: String = ""

    var trancheB: String = ""

    var transferDataS: String = ""

    init() {

    }

    init(json: JSON) {
        self.tokenStandardS = json["tokenStandardS"].intValue
        self.tokenStandardB = json["tokenStandardB"].intValue
        self.tokenStandardFee = json["tokenStandardFee"].intValue
        self.trancheS = json["trancheS"].stringValue
        self.trancheB = json["trancheB"].stringValue
        self.transferDataS = json["transferDataS"].stringValue
    }

    func toJson() -> JSON {
        return JSON()
    }
}
